### ISSUE TITLE
Pico Fx1N PLC

### ISSUE THREAD

**Issue by ismailamet** [suratin27/Pico-PLC-FX1N#3]
Hello, raspberry pi pico fx1n plc does not output in counter value comparison commands.

**Comment by erolcum** [comment]
@ismailamet you are correct.. I will try https://github.com/suratin27/PicoPLC. Did anybody try PicoPLC ?
